packages/core/lib/modules/accounts.js

```diff
@@ -2,63 +2,126 @@ import {WalletRpc} from '../client/wallet-rpc';
 import {UtilCryptoRpc} from '../client/util-crypto-rpc';
 import {KeyringRpc} from '../client/keyring-rpc';
 import {ApiRpc} from '../client/api-rpc';
-import {Wallet} from './wallet';
-import { Errors } from '../errors';
-
-export type KeypairType = 'sr25519' | 'ed25519' |  'ecdsa';
+import {Wallet, WalletEvents} from './wallet';
+import {Errors} from '../errors';
+import {KeypairType, WalletDocument, DocumentType} from '../types';
+import {EventManager} from './event-manager';
 
 export type Account = {
-  id: string;
-  name: string;
-  type: KeypairType;
-  address: string;
-}
+  id: string,
+  name: string,
+  type: KeypairType,
+  address: string,
+  correlations: any[],
+};
+
+export const AccountsEvents = {
+  loaded: 'loaded',
+  accountCreated: 'account-added',
+  accountUpdated: 'account-updated',
+  accountRemoved: 'account-removed',
+};
 
 // TODO: Add events to accounts
```

Comment:
you're adding event manager, etc. in this change. Is this TODO still relevant?

Reply:
fixed